README.md

```diff
@@ -1,32 +1,26 @@
 [![Build Status](https://travis-ci.org/mitodl/micromasters.svg?branch=master)](https://travis-ci.org/mitodl/micromasters) [![codecov](https://codecov.io/gh/mitodl/micromasters/branch/master/graph/badge.svg)](https://codecov.io/gh/mitodl/micromasters)
 
 # MicroMasters
-Portal for learners and course teams to access MITx MicroMasters programs
+Portal for learners and course teams to access MITx MicroMasters programs.
 
-## Major Dependencies
-- Docker
-  - [Download from Docker website](https://docker.com/). Follow the OS X or Linux instructions depending on your OS.
-- [Virtualbox](https://www.virtualbox.org/wiki/Downloads) to run edX locally
-- _(OSX only)_ Node/NPM, and Yarn
-  - OSX recommended install method: [Installer on Node website](https://nodejs.org/en/download/)
-  - No specific version has been chosen yet.
+# Initial setup
 
-## (OSX only) Getting your machine Docker-ready
+MicroMasters mostly follows the same [initial setup steps outlined in the common ODL web app guide](https://github.com/mitodl/handbook/blob/master/common-web-app-guide.md).
+Run through those steps in order with the following changes:
 
-#### Create your docker container:
+1. After [creating the database models](https://github.com/mitodl/handbook/blob/master/common-web-app-guide.md#3-create-data-structures), you should create the Elasticsearch indices via management command:
```

Comment:
I don't see a `3-create-data-structures` section in the common guide, did you mean to link to https://github.com/mitodl/handbook/blob/master/common-web-app-guide.md#3-create-database-tables-from-the-django-models ?

Reply:
Right you are. I updated that section title and didn't update the link